addons/pos_self_order/static/src/app/services/self_order_service.js

```diff
@@ -141,6 +141,13 @@ export class SelfOrder extends Reactive {
         });
     }
 
+    get selfService() {
+        const presets = this.models["pos.preset"].getAll();
+        return this.config.use_presets && presets.length > 1
+            ? this.currentOrder?.preset_id?.service_at
+            : this.config.self_ordering_service_mode;
```

Comment:
should we have the field self_ordering_service_mode if we have presets ? As we use it only when we have 1 preset, we could use the field `service_at` of the preset couldn't we ?